docs/dev/libvirt-howto.md

```diff
@@ -52,6 +52,23 @@ polkit.addRule(function(action, subject) {
 EOF
 ```
 
+### Enable IP Forwarding
+
+Libvirt creates a bridged connection to the host machine, but in order for the
+network bridge to work IP forwarding needs to be enabled. To enable it for the
+current boot:
+
+```
+sudo sysctl net.ipv4.ip_forward=1
```

Comment:
We don't talk about privilege escalation here, since there are many ways to do that.  Can we use:

    ```sh
    sysctl net.ipv4.ip_forward=1
    ```

here?  That also gives us shell syntax highlighting, for what it's worth in this one-liner ;).